tests/Controllers/DashboardControllerTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using staffinfo.divers.Controllers;
+using Xunit;
+
+namespace staffinfo.divers.tests.Controllers
+{
+    public class DashboardControllerTests
+    {
+        [Fact]
+        public void Index_GivenValidInput_ShouldSuccessfullyOpenIndexView()
+        {
+            // Arrange
+            var dashboardController = new DashboardController();
+
+            // Act
+            var result = dashboardController.Index() as ViewResult;
+
+            // Assert
+            Assert.NotNull(result);
```

Comment:
You also need to verify the page name

Reply:
If Index returns null, then check for null ?